cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -378,13 +423,15 @@ func (oi *OVFImporter) Import() error {
 	if err := oi.paramValidator.ValidateAndPopulate(oi.params); err != nil {
 		return err
 	}
-	if err := oi.importDiskFilesToImages(); err != nil {
-		oi.imageDeleter.DeleteImagesIfExist(oi.images)
+	if err := oi.importDisksVMDKFiles(); err != nil {
+		oi.resourceDeleter.DeleteImagesIfExist(oi.images)
+		oi.resourceDeleter.DeleteDisksIfExist(oi.disks)
 		return err
 	}
 	if err := oi.createWorkerForFinalInstance().Run(map[string]string{}); err != nil {
 		oi.Logger.User(err.Error())
-		oi.imageDeleter.DeleteImagesIfExist(oi.images)
+		oi.resourceDeleter.DeleteImagesIfExist(oi.images)
```

Comment:
Could you clarify why do we need to delete images? I thought that they wouldn't be created at all. Do I miss anything?

Reply:
But still we create image for the boot disk, in case of failure I think we need to delete it.